### PR TITLE
add 4 CVEs

### PR DESCRIPTION
**Pull request by CERT-VDE**

### CODE REVIEW

File: 2022/30xxx/CVE-2022-30308.json [hunk 1]
```diff
@@ -1,18 +1,238 @@
 {
```
Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'references' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30308.json:1]: 'description' is a required property

File: 2022/30xxx/CVE-2022-30309.json [hunk 2]
```diff
@@ -1,18 +1,238 @@
 {
```
Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'references' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30309.json:1]: 'description' is a required property

File: 2022/30xxx/CVE-2022-30310.json [hunk 3]
```diff
@@ -1,18 +1,238 @@
 {
```
Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'references' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30310.json:1]: 'description' is a required property

File: 2022/30xxx/CVE-2022-30311.json [hunk 4]
```diff
@@ -1,18 +1,238 @@
 {
```
Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'references' is a required property

Comment (cve-team) [inline 2022/30xxx/CVE-2022-30311.json:1]: 'description' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by CERT-VDE** [comment]
Okay, seems like there is no chance to use CVE 5.0 anywhere to submit CVEs :-(

**Pull request closed without merging**